### DISCUSSION TITLE
flow deck v2 issue

### DISCUSSION THREAD

**Discussion by justinleeyang** [bitcraze/discussions#1654] [General]
Hi, 

My flow deck v2 report error, Motion chip id is 0x49:0xB6, so PWM3901 is ok, VL53L1x fail ?  maybe reason ? 
![1731603978902](https://github.com/user-attachments/assets/6b6c6381-9ba6-4b44-a924-338d946300e7)

**Comment by gemenerik** [comment]
Thanks for bringing up the issue with your flow deck. Was the sensor functioning correctly before? Could there have been physical damage from crashing? Are all connections intact?
